packages/core/echo/live-object/src/ref/ref-impl.ts

```diff
@@ -0,0 +1,140 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { Reference } from '@dxos/echo-protocol';
```

Comment:
we have both Ref and Reference?

Reply:
Reference is an internal implementation detail